### ISSUE TITLE
Typings for Typescript

### ISSUE THREAD

**Issue by hisham** [dhershman1/phone-fns#3]
Nice library! Would be nice to have typings for this lib for use in Typescript projects such as Angular apps. Can be added to the library itself or at https://github.com/DefinitelyTyped/DefinitelyTyped.

**Comment by dhershman1** [comment]
Hey! Thanks for the suggestion, I'm not the most familiar with it but I attempted to add a index.d.ts file if you don't mind checking it out for me that'd be cool!

**Comment by hisham** [comment]
Thanks! I'll test it in the coming days with my actual app. 

I did notice a couple things however after reviewing the code: 

1) Instead of `breakdown(phone: string): object`, you can actually define another interface for object such as:

```
interface Breakdown {
  areaCode: string;
  localCode: string;
  lineNumber: string;
  extension: string;
}
```

And then the method would be `breakdown(phone: string): Breakdown`

2) Shouldn't isValid return boolean? If so then method should be `isValid(phone: string): boolean`

Thanks!

**Comment by dhershman1** [comment]
Woops might've been a copy and paste error on number 2 there, and good idea for breakdown! I will certainly do that.

**Comment by dhershman1** [comment]
It looks like the typing were working properly from an IDE and compile stand point, if you notice any problems please don't hesitate to re open this ticket and let me know! Thanks again for the suggestion!

(Typings should be available in the v3.0.0 release which I just published)

**Comment by hisham** [comment]
Thanks, it's confirmed working for me.
ᐧ
